backend/Dockerfile.buildapi

```diff
@@ -0,0 +1,45 @@
+# 0. Generate client code (go & json) from API protocol buffers
+FROM golang:1.13.1-stretch as generator
+ENV KFP_VERSION 1.0.0
+ENV GRPC_GATEWAY_VERSION v1.9.1
+ENV GO_SWAGGER_VERSION v0.21.0
+ENV GOLANG_PROTOBUF_VERSION v1.3.2
+ENV PROTOCOLBUFFERS_PROTOBUF_VERSION v3.11.3
+ENV GRPC_VERSION v1.23.0
+
+# Install protoc.
+RUN apt-get update -y && apt-get install -y jq sed unzip
+ENV PROTOC_ZIP=protoc-3.13.0-linux-x86_64.zip
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/$PROTOC_ZIP
+RUN unzip -o $PROTOC_ZIP -d /usr/ bin/protoc
+RUN unzip -o $PROTOC_ZIP -d /usr/ 'include/*'
+RUN rm -f $PROTOC_ZIP
+ENV PROTOCCOMPILER /usr/bin/protoc
+ENV PROTOCINCLUDE /usr/include/google/protobuf
+
+# Need grpc-gateway source code for -I in protoc command.
+WORKDIR /go/src/github.com
+RUN mkdir grpc-ecosystem && cd grpc-ecosystem && git clone --depth 1 --branch $GRPC_GATEWAY_VERSION https://github.com/grpc-ecosystem/grpc-gateway.git
+RUN mkdir grpc && git clone --depth 1 --branch $GRPC_VERSION https://github.com/grpc/grpc-go 
+
+# Install protoc-gen-rpc-gateway && protoc-gen-swagger.
+RUN cd grpc-ecosystem/grpc-gateway && GO111MODULE=on go mod vendor
+RUN GOBIN=/go/bin go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+RUN GOBIN=/go/bin go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+
+# Need go-swagger source code for -I in protoc command.
+RUN mkdir go-swagger && cd go-swagger && git clone --depth 1 --branch $GO_SWAGGER_VERSION https://github.com/go-swagger/go-swagger.git
```

Comment:
I think we can download a binary for go-swagger to simplify this: https://github.com/go-swagger/go-swagger/releases/tag/v0.18.0

Reply:
Will update.